flask_server.py

```diff
@@ -67,22 +71,28 @@ def index():
     if request.method == "POST" and is_user_logged_in:
         amount_to_scrape = int(request.form["amountToScrape"]) if request.form["amountToScrape"].isdecimal() else -1
         mode, ids_mode = request.form["mode_dropdown"], request.form["ids_dropdown"]
+        logger.info(f"Starting scraping in mode: {mode}, ids source: {ids_mode}")
         base64_media = start_scrape(settings.get(), folder_path, amount_to_scrape, mode, ids_mode)
+        logger.info("Finished scraping")
         converted_files = convert_media_files(base64_media)
     logged_in_error = request.method == "POST" and not is_user_logged_in
     log_lines = get_log_file_list()
+    logger.info("Loading index")
```

Comment:
It would be useful if you log if the user request is a POST or a GET (as that changes quite a bit)

eg: POST request to /index